src/usr/local/emhttp/plugins/open.files/data.php

```
@@ -0,0 +1,41 @@
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+require_once dirname(__FILE__) . "/include/common.php";
+
+$prefix = "/plugins/open.files/data.php";
+
+$app = AppFactory::create();
```

PHPStan (2.1.15)

Call to static method create() on an unknown class Slim\Factory\AppFactory.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)